server/app.js

```diff
@@ -9,6 +9,44 @@ const request = require('request');
 const bodyParser = require('body-parser');
 const fs = require('fs');
 const htpasswdFilePath = path.resolve(__dirname, '../.htpasswd');
+const prerendercloud = require('prerendercloud');
+const bodybuilder = require('bodybuilder');
+const axios = require('axios');
+// using this instead of ejs to template from the express routes after we fetch object data.
+// because the webpack compiler is already using ejs.
+const Handlebars = require('handlebars');
+const canonicalBase = 'https://collection.barnesfoundation.org';
```

Comment:
maybe make this a `process.env.CANONICAL_BASE || 'https://collection.barnesfoundation.org'`

Reply:
nice - yeah, i just caught that thx